tehom/downloads.py

```diff
@@ -94,8 +95,9 @@ def _unzip_ais(zipfile: Path) -> Tuple[Path]:
         tuple comprising the root of the unzip tree and the specific
         unzipped file of interest
     """
-    # morgan
-    pass
+    with ZipFile(f"{zipfile}", "r") as zipObj:
+        zipObj.extractall(_persistence.AIS_TEMP_DIR)
+    return _persistence.AIS_TEMP_DIR, zipfile.parts[-1]
```

Comment:
First return value: We want to return the root of the unzip tree, not the folder that it's in.

Second return value: `zipfile` is a path to the zipped file.  We want to return the unzipped file's path.

Reply:
how is this?

with ZipFile(f"{zipfile}", "r") as zipObj:
      zipObj.extractall(_persistence.AIS_TEMP_DIR)
  return _persistence.STORAGE, _persistence.AIS_TEMP_DIR / zipfile.parts[-1]

Reply:
^ ignore that